### PR TITLE
[Form Container] Add ability to configure the method from the dialog

### PR DESCRIPTION
**Pull request by brandonmaynard**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | N/A 
| Patch: Bug Fix?          | No
| Minor: New Feature?      | Yes
| Major: Breaking Change?  | No
| Tests Added + Pass?      | Yes
| Documentation Provided   | No - new dialog field
| Any Dependency Changes?  | No
| License                  | Apache License, Version 2.0

The Sling Model for the form container component supports an authored configuration for the form's method, but currently it is only set to POST as its default value. This change adds a dropdown to the dialog to allow the user to change the method to GET if they desire. I am not sure if this was omitted purposely or just a minor oversight.

### CODE REVIEW

**Comment by richardhand** [comment]
Thanks @brandonmaynard, I have created #448 for tracking this. I will check with the team if there was a reason for not allowing it as a configuration in the dialog.

File: content/src/content/jcr_root/apps/core/wcm/components/form/container/v1/container/_cq_dialog/.content.xml [hunk 1]
```diff
@@ -82,6 +82,25 @@
                                 name="./redirect"
                                 rootPath="/content"
                                 wrapperClass="hide cmp-redirect-selection"/>
+                            <method
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/form/select"
+                                defaultValue="POST"
+                                fieldLabel="Form Method"
```
Comment (richardhand) [inline content/src/content/jcr_root/apps/core/wcm/components/form/container/v1/container/_cq_dialog/.content.xml]: We can simply name this `Method` and can we also add a field description?

**Comment by brandonmaynard** [comment]
Thanks @richardhand! Just let me know what the outcome is so I can adjust for it and make the requested changes/remove this PR.

**Comment by gabrielwalt** [comment]
Hi @brandonmaynard, let's discuss this a bit more in detail on #448.

**Pull request closed without merging**